src/guides/v2.3/frontend-dev-guide/theme-best-practice.md

```diff
@@ -31,4 +31,9 @@ We recommend using the following best practices when developing themes:
    <a href="#"><?= __('Click to download'); ?></a>
    ```
 
+1. Make use of the mobile first approach when inheriting blank or luma themes which has already been followed in these themes.
+1. Mangento have a set of [coding stadrads]({{ page.baseurl }}/coding-standards/bk-coding-standards.html) for both back-end and front-end technologies, Do refer and follow them when needed.
+1. Do not repeat a work, instead create a class or method and call them when needed.
```

Comment:
I have doubts regarding this point. If we are talking about PHP classes/methods, they are not supposed to be a part of a theme. They may, but not in the classic Magento Theme definition. So, I would suggest to remove it.

Reply:
Yes, that is correct. will change that. thanks.